_stdlib/__build.dm

```diff
@@ -112,7 +112,7 @@ var/global/ci_dm_version_minor = "100"
 #define BUILD_TIME_FULL "2009-02-13 18:31:30"
 #define BUILD_TIME_YEAR 2053
 #define BUILD_TIME_MONTH 01
-#define BUILD_TIME_DAY 01 //SET ME TO 13 TO TEST YOUR ASS_JAM CONTENT!!
+#define BUILD_TIME_DAY 13 //SET ME TO 13 TO TEST YOUR ASS_JAM CONTENT!!
```

Comment:
You don't need to include changes to this file

Reply:
Fixed.